### PR TITLE
Add a ton of documentation of things we learned over IAP

### LINKED ISSUE #63: Write a guide on "what to do in order to officially launch game"
**Make sure that whatever guide we use is actually correct.** (Suggestions: Do a dry run? Think over the steps carefully?)

Then put them in a reasonable place for docs. GH readme of game? Dir manual? Both?
Perhaps best: a link to the dir manual, which is shown in the GH readme?

Also, perhaps include some steps that are needed before game release (for posterity). For example, examplefuncsplayer is easy to miss.

The steps that I'm aware of, _perhaps incomplete_ and using bc23 as an example :
- Before game release: Submit examplefuncsplayer to Teh Devs team.
- before game release: Commit your specs to master.
- During game release: Make battlecode23 and battlecode23-scaffold repos public.
- Change IS_PUBLIC to TRUE in .github/workflows/release.yml on GitHub. This will make our systems release the game publicly. [[link](https://github.com/battlecode/battlecode23/blob/master/.github/workflows/release.yml)]
- Release version 1.0.0 on GitHub. [[link](https://github.com/battlecode/battlecode23/releases/new)]
-  once GitHub reports that the release was successful, then on admin interface, change BOTH the public and saturn versions to 1.0.0 [[link](https://api.battlecode.org/admin/episodes/episode/bc23/change/)]

### LINKED ISSUE #527: Add how-to docs for admin viewing submissions and scrimmages

### LINKED ISSUE #546: docs about submission freeze / deadline and unfreeze time tweaks
writing as an issue cuz i'm forgetful and want to be organized

all submission deadlines should be 30 seconds past hour
all unfreezes should be, say, 2 hours after start of stream

### LINKED ISSUE #681: Add docs for general how-to backend administration / tech support operations and instructions
See `backend/docs/operations.md` for where this is started and very loosely sketched out.

Can be done while we're teaching the new devs anyways, to save some duplicated effort.

### PR DESCRIPTION
**Pull request by n8kim1**
Resolves #63, resolves #681, resolves #546, resolves #527

Nathan read over like all of webinfra-related slack of all of fall and IAP 2024 and here is what came out.
This includes docs for releasing a game, among many others.

### CODE REVIEW

**Review by acrantel — DISMISSED** [review]
lgtm! ty

File: docs-general/operations.md [hunk 1]
```diff
@@ -1,14 +1,180 @@
...
+  - “admin” will be the only member of Reference Player.
+  - On api.battlecode.org/admin, set the team type to Invisible. you don’t want anyone to try to scrim against it yet
+  - I forgot “admin”’s password but you can reset it at api.battlecode.org/admin. The password should be “good” because the account has at least as many permissions as yours, if not more.
+  - NB! Teams with no members will get deactivated. Your personal account should stay on Teh Dev team.This is why “admin” exists: so that Reference Player can exist without needing you to stay on it.
+
+#### Pre-release testing
+
+- **Test that a game can be played on the website!** Create a couple teams, submit some bots, queue up a match, etc.
+- Also, **test that both clients -- web-based and native -- work!** Grab the replay from that match and load it in.
+  - (The native client, especially its deploy, are sort of )
```
[the remark this answers is not in the document]
Comment (n8kim1) [reply docs-general/operations.md:71]: ....lol idk

**Review by acrantel — DISMISSED** [review]
lgtm pending one minor comment

File: frontend2/docs/onboard.md [hunk 2]
```diff
@@ -56,6 +56,8 @@ We also use Routes, which is React's concept of translating URLs and paths into
 
 Contexts are a way to store and share information across React components. For example, we can store info about the user in a context, and can pass this to the Typescript of React files to work with it. Learn more here: https://react.dev/learn/passing-data-deeply-with-context.
 
+For deeper info about specific topics we love to use, check out https://tkdodo.eu/blog.
```
[the remark this answers is not in the document]
Comment (n8kim1) [reply frontend2/docs/onboard.md:59]: at first yeah
But, Lowell kept linking posts from the blog, touching on all topics

[the remark this answers is not in the document]
Comment (n8kim1) [reply frontend2/docs/onboard.md:59]: "where I write about all things React, TypeScript and of course react-query."

and we use all of those so I'll leave these here

**Comment by n8kim1** [comment]
@acrantel found another doc to write about, so there's significantly more to review, sorry. But at least this PR is finally not draft anymore

File: docs-general/onboard.md [hunk 3]
```diff
@@ -31,33 +31,54 @@ This is a rough depiction of how webinfra code runs in practice / in deployment.
...
+
+Our compile and execute runners are deployed and run through Google Cloud's _Compute Engine_ which allows you to create and run arbitrary virtual machines (VMs). Our virtual machines are deployed via Docker, an system for constructing VMs with any content and running those VMs on a variety of operating systems. For example, we can use Docker to install Django and automatically run your code. Learn more at [their page](https://www.docker.com).
+
+Builds (the process of going from source code to things running on deployed machines) are done via _Cloud Build_. We configure Cloud Build to run in response to various "triggers", and we specify our triggers to be new tag created on a repository.
+
+To specify a commit to be deployed, on GitHub, assign a _tag_ to that commit. In particular, give the tag a certain prefix (see previous tags for examples). Cloud Build will see the tag, notice it matches a configured regex pattern, use the Docker information in your repo, and deploy what is specified by your tag!
+
+All our configuration about deploys is specified via _Terraform_, an infrastructure-as-code service. See the `deploy` file for more.
```
Comment (acrantel) [inline docs-general/onboard.md:58]: ty! having deploy instructions is awesome

**Review by acrantel — APPROVED** [review]
:shipit:

File: docs-general/onboard.md (hunk 3 above)
Comment (n8kim1) [reply docs-general/onboard.md:58]: yaaa it rlly is
you should also thank whoever wrote these -- ie you and jerry

**Pull request merged**